mobile/examples/super_resolution/app/src/main/java/ai/onnxruntime/example/superresolution/SuperResPerformer.kt

```diff
@@ -0,0 +1,61 @@
+package ai.onnxruntime.example.superresolution
+
+import ai.onnxruntime.OnnxJavaType
+import ai.onnxruntime.OrtSession
+import ai.onnxruntime.OnnxTensor
+import ai.onnxruntime.OrtEnvironment
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import java.io.InputStream
+import java.nio.ByteBuffer
+import java.util.*
+
+internal data class Result(
+    var outputBitmap: Bitmap? = null
+) {}
+
+internal class SuperResPerformer(
+    private val ortSession: OrtSession?,
+    private val result: Result?
+) {
+
+    fun analyze(inputStream: InputStream) {
+        // Step 1: convert image into byte array (raw image bytes)
+        val rawImageBytes = inputStream.readBytes()
+
+        // Step 2: get the shape of the byte array and make ort tensor
+        val shape = longArrayOf(rawImageBytes.size.toLong())
+        val env = OrtEnvironment.getEnvironment()
```

Comment:
should we just use a passed in OrtEnvironment? there is already one used to create the OrtSession.